packages/components/molecules/f-media-element/stories/MediaElement.stories.mdx

```diff
@@ -0,0 +1,182 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs';
+import MediaElement from '../src/components/MediaElement.vue';
+import { ALIGN, FONT_SIZE } from '../src/constants';
+
+<Meta
+    title="Components/Molecules/Media Element (f-media-element)"
+    component={MediaElement}
+    argTypes={{
+        textSize: {
+            control: { type: 'select', options: [...Object.entries(FONT_SIZE).map(([, value]) => value)] },
+            defaultValue: 'md'
+        },
+        contentAlign: {
+            control: { type: 'select', options: [...Object.entries(ALIGN).map(([, value]) => value)] },
+            defaultValue: 'left'
+        },
+        imageAlign: {
+            control: { type: 'select', options: [...Object.entries(ALIGN).map(([, value]) => value)] },
+            defaultValue: 'center'
+        }
+    }}
+/>
+
+# Media Element
+
+This document describes the `f-media-element` and how to implement it within a page. The `f-media-element` can be
+displayed in different ways with a few props affecting how its displayed.
+
+## Default
+
+Below is the default state of `f-media-element`
+
+export const MediaElementComponent = (args, { argTypes }) => ({
+    components: { MediaElement },
+    props: Object.keys(argTypes),
+    template: '<media-element v-bind="$props" />'
+});
+
+<Canvas>
+    <Story
+        name="Default"
+        args={{
+            title: 'Example Title',
+            text: 'Example description text to show what the body text looks like.',
+            altText: 'Test alt text',
+            imageUrl: 'https://via.placeholder.com/200',
+        }}
+    >
+        {MediaElementComponent.bind({})}
+    </Story>
+</Canvas>
+
+## Text Size
+
+We can adjust the text size of the content block by changing the `textSize` prop. This can be set to *five* different
+sizes adjusted with a string passed to the prop.
+
+**XX Large** `xxl`
+<Story
+    name="Text Size XXL"
+    args={{
+        title: 'Example Title',
```

Comment:
Not sure exactly if this is possible in mdx, but would be good to have the common values abstracted into an object such that:

```
const commonArgs = {
        title: 'Example Title',
        text: 'Example description text to show what the body text looks like.',
        altText: 'Test alt text'
}
```

And then when adding the Story using:

```
<Story
    name="Text Size XL"
    args={{
        ...commonArgs,
        imageUrl: 'https://via.placeholder.com/175',
        textSize: 'xl'
    }}
>
    {MediaElementComponent.bind({})}
</Story>
```

Reply:
yes we can do that by doing `export` in front of the const